tests/rules/no-invalid-properties.test.js

```diff
@@ -592,5 +618,134 @@ ruleTester.run("no-invalid-properties", rule, {
 				},
 			],
 		},
+		{
```

Comment:
CSS variables `--my-color` is not case sensitive, can we please add a few cases to verify the rule is working as expected for them?